relevanceai/operations_new/dataset_ops.py

```diff
@@ -237,6 +237,14 @@ def vectorize_text(
             output_fields=output_fields,
             **kwargs,
         )
+
+        for i, field in enumerate(fields):
+            self.update_field_children(
+                field=field,
+                field_children=[ops.output_fields[i]],
+                category="vectorize-text",  # Should this be the workflow ID
+                metadata={},
```

Comment:
Shall we add the info here according to config?

Reply:
probably not necessary for front-end, only low end users such as ourselves will understand model config stuff

Reply:
Hmm - I think we might have to just to provide users the flexibility. I'll try to re-borrow some of Joseph's code from before.